examples/ai-core/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "types": ["node"],
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
why was this necessary>

Reply:
sorry just added comment below, will make sure to add them before posting